packages/sdk-ruby/src/workstream_protocol/lib/workstream_protocol/Onboarding_pb.rb

```diff
@@ -89,6 +89,7 @@
       optional :home_email, :string, 31
       optional :home_phone, :message, 32, "Workstream.Protocol.Common.Phone"
       optional :home_address, :message, 33, "Workstream.Protocol.Common.Address"
+      optional :mobile_phone, :message, 34, "Workstream.Protocol.Common.Phone"
```

Comment:
did you try the API to see if they accept the same value for home phone and mobile phone since both maps to the same value?

home phone wan't mapped for ADP and Paychex.

Reply:
As you can see in this PR, I mapped workstream's phone to the `mobile phone` only.

https://github.com/helloworld1812/workstream-backend/pull/2806/files#diff-50785991475a61bc3786b5430ceaaea489f4000bf1806bd11ad3d965495c4fb5R30